src/main/java/mate/academy/dao/MySqlBookDao.java

```diff
@@ -0,0 +1,123 @@
+package mate.academy.dao;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.lib.ConnectionUtil;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+import mate.academy.model.DataProcessingException;
+
+@Dao
+public class MySqlBookDao implements BookDao {
+    @Override
+    public Book create(Book book) {
+        String sql = "INSERT INTO book (title, price) VALUES (?,?)";
```

Comment:
fix variable name for all methods 
```suggestion
        String query = "INSERT INTO book (title, price) VALUES (?,?)";
```